app/main.py

```diff
@@ -2,0 +1,7 @@
+def copy_file(command: str = "cp") -> None:
+    with open(f"{command} file.txt", "r") as r_file:
```

Comment:
Code Style: Use one style of quotes in your code. Here you're using both single and double quotes, which can make your code less readable. Choose one style and stick to it. Double quotes are preferable.